src/psyclone/f2pygen.py

```diff
@@ -1042,6 +1045,7 @@ class DeclGen(BaseDeclGen):
     :param initial_values: Initial value to give each variable.
     :type initial_values: list of str with same no. of elements as \
                           entity_decls.
+    :param bool private: whether this declaration has the PRIVATE attribute.
```

Comment:
Seeing as you're on a declaration tidying spree, could you please get rid of the capital in "Initial" on line 1045 and  remove the full stop at the end of the type declarations.

Reply:
Could you also say, defaults to False as you do for TypeDecl.

Reply:
Done and done.